src/twisted/web/http.py

```diff
@@ -1085,6 +1089,11 @@ def write(self, data):
                             category=DeprecationWarning, stacklevel=2)
                         # Backward compatible cast for non-bytes values
                         value = networkString('%s' % (value,))
+                    if (self.method == b"CONNECT" and self.code//100 == 2 and
+                        name == b"Content-Length"):
+                        log.msg("Warning: Removing Content-Length header in"
```

Comment:
It looks like the logging style in this module has changed in the interim.  Can you make this `self._log.info(...)` instead (I would almost say something about not putting "Warning" there but I see that other code in this module emits info messages that start with "Warning" so ...)